### ISSUE TITLE
KeyError: "readableProduct"

### ISSUE THREAD

**Issue by bega1Q** [eggplants/getjump#271]
When I try to use the script to download from Comic-Days, I get the error KeyError: 'readableProduct'.
I'm leaving all the logs in case it helps.

```
C:\Users\begab\Downloads\Respaldo\Descargar mangas\getjump-master> jget -d C:\Users\begab\OneDrive\Escritorio\Raws_mangas https://comic-days.com/episode/2550689798905518166

get: https://comic-days.com/episode/2550689798905518166
Traceback (most recent call last):
  File "C:\Users\begab\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\begab\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\begab\AppData\Local\Programs\Python\Python310\Scripts\jget.exe\__main__.py", line 7, in <module>
  File "C:\Users\begab\AppData\Local\Programs\Python\Python310\lib\site-packages\getjump\main.py", line 185, in main
    get_one(args)
  File "C:\Users\begab\AppData\Local\Programs\Python\Python310\lib\site-packages\getjump\main.py", line 163, in get_one
    _, prev_title, ok = g.get(
  File "C:\Users\begab\AppData\Local\Programs\Python\Python310\lib\site-packages\getjump\getjump.py", line 101, in get
    j = res.json()["readableProduct"]
KeyError: 'readableProduct'
PS C:\Users\begab\Downloads\Respaldo\Descargar mangas\getjump-master>
```

**Comment by eggplants** [comment]
What version are you using? Your problem has fixed in #265 and v2.6.0.

In my env it works without errors:

```shellsession
$ getjump -V
2.6.2
$ python -V
Python 3.13.0
$ getjump https://comic-days.com/episode/2550689798905518166
get: https://comic-days.com/episode/2550689798905518166
  Downloading... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% ( 15/15 pages ) remain: 0:00:00 spent: 0:00:02
  Saving...      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% ( 15/15 pages ) remain: 0:00:00 spent: 0:00:00
saved: しゅごキャラ！ ジュエルジョーカー/第１話（４）
done.
```

**Comment by bega1Q** [comment]
Resolved, I thought I was running the script from the file, but I forgot that I had also installed it with 'pip install getjump'.

I just used 'pip install --upgrade getjump' and it worked.
